packages/@aws-cdk/aws-cloudfront/test/web_distribution.test.ts

```diff
@@ -426,8 +426,7 @@ nodeunitShim({
     const stack = new cdk.Stack();
     const sourceBucket = new s3.Bucket(stack, 'Bucket');
 
-    const lambdaFunction = new lambda.SingletonFunction(stack, 'Lambda', {
```

Comment:
Had to move from a `SingletonFunction` to `Function` because `SingletonFunction` has no `addVersion()` it's a `FunctionBase`. It now has a `_checkEdgeCompatibility()` but no `addVersion()` yet, for this PR?